### PR TITLE
ConsoleReader

### PR DESCRIPTION
**Pull request by Stanislava88**
Task1 for reading different values from console

### CODE REVIEW

File: src/console/Console.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package console;
+
+import java.util.Scanner;
+
+/**
+ * @author Stanislava Kaukova(sisiivanovva@gmail.com)
+ */
+public class Console {
+    private Scanner value = new Scanner(System.in);
```
Comment (mgenov) [inline src/console/Console.java]: It's scanner, not value.

File: src/console/Console.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    }
+
+    public float readFloat() {
+        float number = value.nextFloat();
+        return number;
+    }
+
+    public String closeScanner() {
+        String msg = "The Scanner is closed!";
+        value.close();
```
Comment (mgenov) [inline src/console/Console.java]: In some rare cases closing of stdin/stdout is good idea, but this is not the case, so the method could be removed.

File: src/console/Console.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        return string;
+    }
+
+    public int readInt() {
+        int num = value.nextInt();
+        return num;
+    }
+
+    public char readChar() {
+        char character = value.next().charAt(3);
```
Comment (mgenov) [inline src/console/Console.java]: What is magical in 3 ?

**Comment by Stanislava88** [comment]
Done

**Comment by mgenov** [comment]
LGTM !

**Pull request merged**